### PR TITLE
follow Code and Syntax Highlighting

### PR DESCRIPTION
**Pull request by Sibiraj-S**
follow Code and Syntax Highlighting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,23 +8,25 @@ It calculates how many pages it needs/can display for user to select from based
...
 
-```
+```bash
 npm install ngx-paginate --save
 ```
 
 Import module
-```
+
+```bash
```
Comment (slavede) [inline README.md]: In fact, this is also a Typescript snippet, can you change that to TS? Other than that, tnx for the PR

Comment (Sibiraj-S) [reply README.md]: since you included. `...` in the code. that is an invalid ts. So that I purposely mentioned bash there.

Comment (slavede) [reply README.md]: But you did remove `. . .`. Didn't you? :)

Comment (Sibiraj-S) [reply README.md]: haha.😅 I didn't notice that

Comment (Sibiraj-S) [reply README.md]: I deleted the forked repo. You can merge and you can edit that yourself 😄

Comment (slavede) [reply README.md]: done xD

**Pull request merged**